.test-infra/terraform/google-cloud-platform/utility-cluster/strimzi-contoller.tf

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "helm_release" "strimzi-helm-release" {
+  name       = "strimzi"
+  namespace  = "strimzi"
+  create_namespace = true
+  repository = "https://strimzi.io/charts/"
+  chart      = "strimzi-kafka-operator"
+  version = "0.40.0"
+  
+  atomic = "true"
+  timeout = 500
+
+  set {
+    name  = "watchAnyNamespace"
+    value = "true"
+  }
+  depends_on = [ module.gke.google_container_cluster ]
+}
+
```

Comment:
This could be in its own module separate from the GKE cluster provisioning.

Reply:
Yes it is possible to put it in its own module but the idea behind the `utility-cluster` folder is to use the GKE module and install everything that is needed for that exact purpose via terraform and in one step. It does not make sense for me to separate out a module as there is no intetion to reuse this due to its specific purpose. Other clusters can crate different folders for different purposes. 
Let me know if this is fine and if not ill try to come up with different structure

Reply:
In my experience, I find co-mingling resource provisioning with Kubernetes workload provisioning in the same terraform module to lead to problems in the future. I personally would like to see it in a separate module. I'm more than willing to defer to another Apache Beam committer's opinion, if they think the co-mingling design is ok. In summary, my design preference is:
1) separate GKE provisioning module - a version controlled tfvars file in the existing `.test-infra/terraform/google-cloud-platform/google-kubernetes-engine` folder could work
2) separate folder responsible for provisioning the strimzi cluster